tests/lib/rules/no-unguarded-globals.js

```diff
@@ -504,6 +504,72 @@ ruleTester.run('no-unguarded-globals', rule, {
       errors: [{
         message: DOCUMENT_MESSAGE
       }]
+    },
+    {
+      code: `
+        import environment from 'ember-stdlib/utils/environment';
+
+        export default Ember.Component.extend({
+          shortCircuit() {
+            if (environment.isBrowser() && window.localStorage) {
+              return window.localStorage;
+            }
+          }
+        });`,
+      parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+      }
+    },
+    {
+      code: `
+        import environment from 'ember-stdlib/utils/environment';
+        const { isBrowser } = environment
```

Comment:
Can we add a test to ensure that aliases are still being captured and followed?

```js
const { isBrowser: isB } = environment; 
```

Reply:
This doesn't work in master...

```js 
 1) no-unguarded-globals valid 
        import environment from 'ember-stdlib/utils/environment';
        const { isBrowser: isB } = environment;

        export default Ember.Component.extend({
          isInBrowser() {
            if (isB()) {
              return window.location;
            }
          }
        });:

      AssertionError [ERR_ASSERTION]: Should have no errors but had 1: [ { ruleId: 'no-unguarded-globals',...
```

Reply:
OK, we should fix it. We have utilities that handle this in `ember-best-practices`, but because that repo was created after this one it's likely they weren't back-ported.

Reply:
I lied, it doesn't look like we support aliases. We should put that in our TODOs.